app/build.gradle.kts

```diff
@@ -6,6 +6,10 @@ android {
     namespace = "com.example.pygmyhippo"
     compileSdk = 34
 
+    dataBinding {
```

Comment:
curious, what does this do?

Reply:
I think this may have been added automatically, I am not sure becuase I don't think I wrote that. https://stackoverflow.com/q/59390492 apparently it's old, I'll look into seeing what happens if this is changed to the new way.